test/servico/cst04Test.java

```diff
@@ -60,7 +60,7 @@ public void tearDown() {
     public void testeEfetuarAlocação_Confirmar() throws ParseException
     {
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Administrador", "123456"));
+        assertTrue(loginService.verificarUsuarioESenha("Administrador", "123456"));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['method def' child have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/calebepb/substituicao/file/2777960756/issues/source?bid=3364462&fileBranchId=3364484#l63)